client/layout/index.jsx

```diff
@@ -403,14 +403,14 @@ export default withCurrentRoute(
 				// hide the masterBar until the section is loaded. To flicker the masterBar in, is better than to flicker it out.
 				! sectionName ||
 				( ! isWooCoreProfilerFlow && [ 'signup', 'jetpack-connect' ].includes( sectionName ) );
+			const isDesktop = isWithinBreakpoint( '>782px' );
```

Comment:
It would be better to use `subscribeIsWithinBreakpoint`. Sometimes the master is still there after resizing.

Moreover, I assume we have to support `>=782px` as the sidebar is open when the screen width is `782px` and the same for lots of CSS (or we have to change ALL to 783px for the larger screen at once 😅)

Reply:
nice! thanks for the feedback `subscribeIsWithinBreakpoint` should fix it.